nav2_mppi_controller/src/path_handler.cpp

```diff
@@ -55,19 +57,38 @@ PathRange PathHandler::getGlobalPlanConsideringBounds(
       return euclidean_distance(global_pose, ps);
     });
 
-  // Find the furthest relevent point on the path to consider within costmap
-  // bounds
-  const auto * costmap = costmap_->getCostmap();
+  nav_msgs::msg::Path transformedPlan;
```

Comment:
```suggestion
  nav_msgs::msg::Path transformed_plan;
```

that's out of style for the project